src/librustc_mir/borrow_check/mod.rs

```diff
@@ -1966,8 +1956,18 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
             }
         }
 
+        // rust-lang/rust#21232, #54986: during period where we reject
```

Comment:
The "during period" comment is outdated now that the period is over? cc @tmandry @pnkfelix

Reply:
What? #54987 is still open, and as far as I know we are still rejecting attempts to partially initialize... is that not the case? I just now noticed the additional points added by Ralf and others about how that feature would interact with ideas/plans to make things like `(String, !)` a ZST, too... so the comment seems still relevant to me. But I haven’t tried anything in the playpen yet...

Reply:
Ah; on re-reading the comment it seems its not about the warning we emitted back then but which we turned into a hard error. Makes sense now.